src/haskell/install.sh

```diff
@@ -52,6 +52,16 @@ fi
 # Here's some more examples using it: https://github.com/search?q=org%3Adevcontainers+_REMOTE_USER&type=code
 # We also use /bin/sh as defined in the script hash-bang line instead of $SHELL.
 sudo -iu "$_REMOTE_USER" <<EOF
+  # Add some env vars as discussed in https://github.com/devcontainers-contrib/features/pull/94#issuecomment-1330004657
+  export BOOTSTRAP_HASKELL_NONINTERACTIVE='$BOOTSTRAP_HASKELL_NONINTERACTIVE'
+  export GHCUP_USE_XDG_DIRS='$GHCUP_USE_XDG_DIRS'
```

Comment:
Let's add this to the `~/.bashrc` file to make everything happy for the remote user.

```suggestion
  export GHCUP_USE_XDG_DIRS='$GHCUP_USE_XDG_DIRS'
  echo 'export GHCUP_USE_XDG_DIRS=$GHCUP_USE_XDG_DIRS' >> ~/.bashrc
```